src/iccpd/src/iccp_cli.c

```diff
@@ -117,7 +117,12 @@ int set_peer_link(int mid, const char* ifname)
                        csm->mlag_id, ifname);
     }
 
-    memset(csm->peer_itf_name, 0, MAX_L_PORT_NAME);
+    memset(csm->peer_itf_name, 0, IFNAMSIZ);
+    if (len > MAX_L_PORT_NAME)
```

Comment:
this also needs to change to check for IFNAMSIZ

Reply:
fixed